### PR TITLE
WeConnect UI templates

### PR DESCRIPTION
**Pull request by JoshuaOndieki**
**What does this PR do?**
Adds UI templates for the WeConnect project.

**Description of Task to be completed?**
The front end UIs should be able to show how all the necessary processes of the WeConnect web app would be performed.

**Tasks achieved in this PR**

- [x] Landing page

- [x] Login page

- [x] User homepage where logged in users are redirected to. Users can manage their businesses and view reviews feed on this page. The add and edit business are modal forms.

- [x] A business profile page where users can view the business info and also write more reviews.

- [x] A business listing page for discovering new businesses and easier search of the listed businesses.

**How should this be manually tested?**
Check out [joshuaondieki.github.io](https://joshuaondieki.github.io) and navigate through the UIs for manual testing. Alternatively, clone this repo and test locally on your computer.

**Resources & Libs**
The material UI by Google was used to accomplish this task. Specifically, I used [Materialize CSS](https://www.materializecss.com)
[This Pivotal Tracker board](https://www.pivotaltracker.com/n/projects/2153376) was used to plan and manage the tasks.

### CODE REVIEW

**Review by moki-daniel — CHANGES_REQUESTED** [review]
- Good branching and commits
- Add more comments on the code for easy understand
- Well arranged code 
- For the work looks okay

**Comment by JoshuaOndieki** [comment]
Thanks, @moki-daniel for reminding me about the comments. Will do so ASAP

**Comment by migot01** [comment]
-Nice concept of design
-Good choice of color

**Comment by menyagah** [comment]
-nice layout. I love the color blending.

**Comment by martxsub** [comment]
I've commented on your repo

File: designs/UI/css/business.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#results{
+    width: 90%;
+    margin: auto;
```
Comment (cosmas28) [inline designs/UI/css/business.css:3]: What do you think about making margin property DRY. I'm not sure, but try to move margin property to a new class/id selector and the nest it with other selectors that neede the property

Comment (JoshuaOndieki) [reply designs/UI/css/business.css:3]: DRY Noted. Will check and group all common props

**Comment by artorious** [comment]
* Good job at Gitflow
* Check on the site's responsiveness on small screens

oj1
oj2

**Comment by cosmas28** [comment]
I'm not sure, but are you using one branch to commit all these changes?? What do you think about creating a branch for every design idea and fixes??

**Comment by JoshuaOndieki** [comment]
@artorious Have you got any specific suggestion to make the design more responsive? Like, replacing X property with Y for component Z

**Comment by JoshuaOndieki** [comment]
@cosmas28 I thought of branches but I've been working on different design ideas simultaneously. This may lead to conflicts if I use more than one branch. Your idea is great :ok_hand: and I will use it for features that are less or not directly related. Probably in the flask API

**Comment by artorious** [comment]
Consider .... Screen width

screenshot from 2018-03-01 08 22 59

**Pull request merged**